assets/js/modules/fontFaceLoader.js

```diff
@@ -0,0 +1,25 @@
+import FontFaceObserver from 'fontfaceobserver';
+import fonts from '../../../fontfaces.json';
+
+if (!CSS.supports('font-display')) {
+    const fontPromises = [];
+
+    fonts.forEach((font) => {
+        const family = font.family.replace('\'', '');
+        const loader = new FontFaceObserver(family, {
+            weight: font.weight,
+            style: font.style
+        });
+
+        const promise = loader.load();
+        fontPromises.push(promise);
+
+        promise.then(() => {
+            document.documentElement.className += ' wf-' + font.family.replace(/\s/g, '') + '-' + font.weight + '-' + font.style;
+        });
+    });
+
+    // Promise.all(fontPromises).then(function () {
```

Comment:
Useful ?

Reply:
Only for debug. Maybe we should trig an event ?
It can be useful if font load size of content can change and some scripts could need to calcul new height of elements.